funcx_sdk/funcx/sdk/client.py

```diff
@@ -599,11 +574,7 @@ def get_endpoint_status(self, endpoint_uuid):
         dict
             The details of the endpoint's stats
         """
-        stats_path = f"endpoints/{endpoint_uuid}/status"
-
-        r = self.get(stats_path)
-
-        # Return the result
+        r = self.web_client.get_endpoint_status()
```

Comment:
Shouldn't `endpoint_uuid` be passed to `get_endpoint_status`?

Reply:
Good catch, yes! I'll fix this as part of a rebase to catch up with the recent changes here.